sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Customized/Models/EventAvailability.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Iot.TimeSeriesInsights
+{
+    // TODO: This class has been customized to suppress a build error that the name of the class is too generic and single worded. We
```

Comment:
let's make sure these suggested swagger changes are included in the PR we have open for this swagger document.
https://msazure.visualstudio.com/One/_git/Azure-IoT-Platform-Specs/pullrequest/3743987?_a=files&path=%2Fiotdps%2Fdataplane%2F2020-12-01%2Fdevice.json&discussionId=64853319

Reply:
The link you pasted is for DPS, not TSI. I don't think there is an open PR for TSI swagger changes. It's something I plan to discuss with them when I onboard them.

Reply:
ah, I got that wrong :)